### DISCUSSION TITLE
IOS: Keyboard hides the input

### DISCUSSION THREAD

**Discussion by 2manoj1** [ionic-team/capacitor#3885] [Q&A]
Keyboard hides the input, on value enter content pushed up, need a solution for angular 10+ with ionic 5.4.1 and capacitor 2.4.2 How will handle this problem on current iOS development?

**Comment by irving-caamal** [comment]
check for [Keyboard API -> setAccessoryBarVisible](https://capacitorjs.com/docs/apis/keyboard#setaccessorybarvisible) which allows you, for example, to add a DONE button, which you can use it to close the keyboard or put the navigation buttons on the top of it.

**Reply by aparajita** [reply]
Add this to capacitor.config.json:

```json
{
  "Plugins": {
    "Keyboard": {
      "resize": "body"
    }
  }
}
```

This will cause input fields to be scrolled into view when focused.

**Reply by 2manoj1** [reply]
Its fixes with ion-content but if we use "div" its not working with ```{
  "Plugins": {
    "Keyboard": {
      "resize": "body"
    }
  }
}```
